app/controllers/EnumeratorExamples.scala

```diff
@@ -1,31 +0,0 @@
-package controllers
-
-import play.api.mvc.Controller
-import play.api.mvc.Action
-import play.api.libs.iteratee.Enumerator
-import play.api.libs.concurrent.Promise
-import play.api.libs.concurrent.Execution.Implicits._
-
-object EnumeratorExamples extends Controller {
```

Comment:
Why delete this controller? 

To provide some context, this is a repo that I use as part of my talk on Composable and Streamable Play Apps ([slides](http://www.slideshare.net/brikis98/composable-and-streamable-play-apps), [video](https://www.youtube.com/watch?v=4b1XLka0UIw)), and this controller and many of the others are a part of that talk.